source/templates.tex

```diff
@@ -1390,16 +1388,14 @@
   T& elem(int i) { return v[i]; }
 };
 \end{codeblock}
-
-\pnum
 The prefix
 \tcode{template}
 \tcode{<class}
 \tcode{T>}
 specifies that a template is being declared and that a
 \grammarterm{type-name}
 \tcode{T}
-will be used in the declaration.
+will be used in the definition.
```

Comment:
But the example doesn't do that. The example should help make things clear, not list unrelated, though un-wrong, statements. Should we maybe delete the sentence?